lib/LDA.py

```diff
@@ -13,12 +13,12 @@ def __init__(self, abstracts, n_factors, n_iterations=5):
         self.n_iterations = n_iterations
 
     def train(self):
-        tf_vectorizer = CountVectorizer(max_df=0.95, min_df=2, stop_words='english', max_features=self.n_items)
-        tf = tf_vectorizer.fit_transform(self.abstracts)
+        term_freq_vectorizer = CountVectorizer(max_df=0.95, min_df=2, stop_words='english', max_features=self.n_items)
```

Comment:
Okay, we can merge it for now, and later on I will try to check if I can replace it with stuff from the DataParser. opened for this #9